eox_tenant/settings/common.py

```diff
@@ -51,6 +51,8 @@ def plugin_settings(settings):
     settings.UTILS_MODULE_BACKEND = 'eox_tenant.edxapp_wrapper.backends.util_h_v1'
     settings.CHANGE_DOMAIN_DEFAULT_SITE_NAME = "stage.edunext.co"
     settings.EOX_TENANT_LOAD_PERMISSIONS = True
+    settings.EOX_TENANT_ENROLLMENTS_BACKEND = 'eox_tenant.edxapp_wrapper.backends.enrollments_h_v1'
```

Comment:
@cocococosti @jfavellar90 I don't know if this applies here but I think this file should point to 
```suggestion
    settings.EOX_TENANT_ENROLLMENTS_BACKEND = 'eox_tenant.edxapp_wrapper.backends.enrollments_i_v1'
```
and the corresponding file should be renamed to that version too, Does that make sense here?

Reply:
I named the files following the example of the other ones, the i in enrollments_i_v1 is for ironwood?

Reply:
Yes, that is the meaning